mobile/src/reducers/organization/organization.epics.ts

```diff
@@ -12,6 +12,8 @@ import { Employee } from './employee.model';
 import { Observable } from 'rxjs/Observable';
 import { loadFailedError } from '../errors/errors.action';
 
+const debounceTime = 10000;
```

Comment:
put a milliseconds somewhere in the name, otherwise it's unclear

Reply:
ok

Reply:
addressed